pkg/kubelet/kubelet.go

```diff
@@ -60,6 +60,7 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/bandwidth"
 	utilErrors "k8s.io/kubernetes/pkg/util/errors"
+	kubeioutil "k8s.io/kubernetes/pkg/util/io"
```

Comment:
The problem here is that we already import ```io``` and ```io/ioutil```. We use
1. ```utilio``` (but it's pretty close to ```ioutil```)
2. ```kioutil```
3. ```kubeio```
From those three I'd prefer ```kubeio```. Please let me know if you want me to change it to something else.